docs/DeveloperGuide.md

```diff
@@ -395,23 +395,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ------------------------------------------ | ------------------------------------------------- | ------------------------------------------------------------ |
 | `* * *`  | user                                       | create a student's profile                        | keep track of my student's information                       |
 | `* * *`  | user                                       | see the profile of a particular student           | get their information                                        |
-| `* * *`  | user                                       | see a list of all my students' profile            |                                                              |
-| `* * *`  | user                                       | get all the emails of the parent of my students'  | email them reminders for payment                             |
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
+| `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
 | `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
 | `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | user                                       | see a list of all the students profile and sessions|                                                             |
+| `* * *`  | user                                       | get all the emails of the parent of my students'  | email them reminders for payment                             |
+| `* * *`  | user                                       | calculate the monthly fees of a particular student| use the information when collecting monthly fees             |
+| `* * *`  | user                                       | get the monthly fees that I would have received for the past 3 months    | see how well I am doing financially   |
+| `* *`    | new user                                   | get a list of commands of the application         | know at a glance what are the features of the application    |
+| `* *`    | potential user                             | see the app populated with sample data            | easily see how the app will look like when it is in use      |
```

Comment:
For line 407, How about changing it to:
As a potential user, I want to **see the app populated with sample data on the first run** so that I can **try using the features easily**

Reply:
Yeap! Sounds good! Have updated in the latest commit.